src/main/java/seedu/address/ui/IterationListItem.java

```diff
@@ -41,7 +43,9 @@ public IterationListItem(Iteration iteration, int displayIndex) {
         date.setText(iteration.getDate().date.toString());
         description.setText(iteration.getDescription().description);
         feedback.setText(iteration.getFeedback().feedback);
-        artImage.setImage(placeholder);
+        String fxImagePath = iteration.getImagePath().path;
+        artImage.setImage(new Image(new File(System.getProperty("user.dir") + "/src/main/resources/"
+                + fxImagePath).toURI().toString()));
```

Comment:
Oh, this is nice, I couldn't figure out how to get out of the src/main/resources, so I put the imgs there. Then I think it better to move all the image data to our /data folder and also set folderpath in the UserPrefs.java file